tools/accuracy/accuracy_checker.py

```diff
@@ -125,28 +163,74 @@ def main():
     test_inputs = {}
     for name, shape in model.get_parameter_shapes().items():
         if args.verbose:
-            print('Parameter {} -> {}'.format(name, shape))
+            print(f'Parameter {name} -> {shape}')
         in_shape = shape.lens()
         in_type = shape.type_string()
-        if not args.fill1:
+        if not args.fill1 and not args.fill0:
             test_input = np.random.rand(*(in_shape)).astype(
                 get_np_datatype(in_type))
-        else:
+        elif not args.fill0:
             test_input = np.ones(in_shape).astype(get_np_datatype(in_type))
+        else:
+            test_input = np.zeros(in_shape).astype(get_np_datatype(in_type))
         test_inputs[name] = test_input
         params[name] = migraphx.to_gpu(migraphx.argument(test_input))
 
     pred_migx = np.array(migraphx.from_gpu(model.run(params)[-1]))
```

Comment:
I think you can just set `offload_copy=True`

Reply:
I tried that originally, but for some reason I was getting errors on larger batch sizes.

Reply:
what were the errors ?

Reply:
I can't seem to reproduce anymore, so I've just updated to have offload_copy=True.